scripts/diff_files

```diff
@@ -0,0 +1,3 @@
+
+./tpls
+./core/src/Cuda/Kokkos_Cuda_Atomic_Intrinsics.hpp
```

Comment:
yes, that is the files of expected diffs.

Reply:
Can you elaborate about the adding the build directory?   What (presumably generated) files were problematic?
I am asking because if these files are being installed we might still want the copyright in them.

Note that this is interesting because my initial reaction was "if you used `git ls-files`" like I suggested that would not be an issue, but it would not help if we really wanted to enforce that all installed files have our copyright

Reply:
It is an interesting question, for now the problem was that it did find stuff like some config files which weren't correct and the way our CI is set up it created a build directory inside the checked out directory.